### DISCUSSION TITLE
PDF Exporter

### DISCUSSION THREAD

**Discussion by EggofTreachery** [chuangcaleb/obsidian-fountain-editor#39] [Ideas]
Haven't found a good way to export directly from obsidian without it messing up or turning the fountain formatting into plain text. A way to export as a PDF with the correct formatting would be really good.

**Comment by chuangcaleb** [comment]
Hey! I also don't really use the export PDF from Obsidian. I've been using the export command from https://marketplace.visualstudio.com/items?itemName=piersdeseilligny.betterfountain and it's working out well for me so far, you could also try https://afterwriting.com/ — it's a manual process for now

**Reply by EggofTreachery** [reply]
Afterwriting worked well! Ty!
